src/service/gnuradio/GnuRadioAcquisitionWorker.hpp

```diff
@@ -371,7 +370,7 @@ class GnuRadioAcquisitionWorker : public Worker<serviceName, TimeDomainContext,
                         auto entries = signalEntryBySink | std::views::values;
                         _updateSignalEntriesCallback(std::vector(entries.begin(), entries.end()));
                     }
-                    _scheduler             = std::make_unique<scheduler::Simple<scheduler::ExecutionPolicy::multiThreaded>>(std::move(*pendingFlowGraph), _threadPool);
```

Comment:
Correct fix w.r.t. API, but `_threadPool` should be removed and the global CPU-processing thread-pool adjusted w.r.t. min/max threads.